soh/src/overlays/actors/ovl_En_Ds/z_en_ds.c

```diff
@@ -164,7 +165,7 @@ void EnDs_OfferOddPotion(EnDs* this, PlayState* play) {
 }
 
 s32 EnDs_CheckRupeesAndBottle() {
-    if (gSaveContext.rupees < 100) {
+    if (GameInteractor_Should(VB_CHECK_RANDO_PRICE_OF_GRANNY, gSaveContext.rupees < 100, NULL)) {
```

Comment:
See the comment below.
```suggestion
    if (GameInteractor_Should(VB_GRANNY_CONSIDER_INSUFFICIENT_RUPEES, gSaveContext.rupees < 100, NULL)) {
```